src/elm/Page/Shop/Summary.elm

```diff
@@ -37,7 +37,7 @@ type Msg
     | ReceiveBuyOffers (Result Http.Error Reservation)
     | ReceiveRecurringPayments (Result Http.Error (List RecurringPayment))
     | SetPaymentSelection PaymentSelection
-    | SetStorePayment Bool
+    | ToggleStorePayment Bool
```

Comment:
"Toggle" betyr at den endrer verdien den allerede har (altså at på linje 137 så ville du gjort `storePayment = not model.storePayment`), men her tar vi jo fortsatt inn verdien den skal sette - med andre ord er "SetStorePayment" korrekt.

Reply:
Fikset!